Source/fsConfig.cs

```diff
@@ -36,6 +36,11 @@ public static class fsConfig {
         private static fsMemberSerialization _defaultMemberSerialization = fsMemberSerialization.Default;
 
         /// <summary>
+        /// Serialize default values?
```

Comment:
What about: "Should default values be included in serialization output?"

Does this impact array/list serialization? In protobuf, if you do not serialize defaults then arrays/lists will be left as null.